requirements.txt

```diff
@@ -1,3 +1,12 @@
-datajoint>=0.13.0
-element-interface>=0.4.0
-plotly
+datajoint==0.14.0
+element_interface==0.5.0
+ipywidgets==8.0.4
+matplotlib==3.7.0
+mkdocs_gen_files==0.4.0
```

Comment:
```suggestion
```

Reply:
This requirement is included in the [Docker image](https://github.com/datajoint/element-calcium-imaging/blob/605f91b97f2d3c594a0e827c7b0eb5d613ae9808/docs/.docker/pip_requirements.txt#L7) that deploys the docs.

Reply:
ohh okay, I'll make the update soon